### PR TITLE
New text/legend for high privacy selection

### PR DESCRIPTION
**Pull request by juanky201271**
A new text-legend to help to the user when change from privacy `normal` to `high`.

### CODE REVIEW

File: app/translations/en.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
   "donation": "Thanks for supporting Zingo!",
   "restarting": "Error connecting to the Server, the App will restart in a few seconds.",
   "change-privacy": "New privacy policy: ",
```
Comment (Oscar-Pepper) [inline app/translations/en.json:16]: can we have the format of this snackbar to be:
New privacy policy: High
Tap in an amount.....

it is all on one line and not easy to read

Comment (Oscar-Pepper) [reply app/translations/en.json:16]: i will PM you a screenshot

File: app/translations/en.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
...
+  "change-privacy-legend": ". Tap in any amount as: 'ZEC -.- - - -' or '$ -.- -' to reveal the value.",
```
Comment (Oscar-Pepper) [inline app/translations/en.json]: i think it is easier to read if it just says "Tap to reveal hidden values"
i find the amount of words and the examples harder to understand personally

File: app/translations/en.json [hunk 3]
```diff
@@ -145,7 +146,7 @@
     "privacys": [
       {
         "value": true,
-        "text": "High Privacy, all the sensitive information is hidden."
+        "text": "High Privacy, all the sensitive information is hidden. You can Tap in any amount as: 'ZEC -.- - - -' or '$ -.- -' to reveal the value."
```
Comment (Oscar-Pepper) [inline app/translations/en.json]: and here:
"High Privacy, all the sensitive information is hidden. Tap to reveal hidden values."
what do you think?

**Review by Oscar-Pepper — CHANGES_REQUESTED** [review]

**Review by Oscar-Pepper — APPROVED** [review]

**Comment by juanky201271** [comment]
I checked that the privacy message is in two lines.

**Pull request merged**